app/uk/gov/hmrc/pegaproofofconceptstubs/models/ResponseModel.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.pegaproofofconceptstubs.models
+
+import play.api.libs.json.{Json, OFormat}
+
+import scala.util.Random
+
+final case class ResponseModel(ID: String, nextAssignmentID: String, nextPageID: String, pxObjClass: String)
+
+object ResponseModel {
+
+  @SuppressWarnings(Array("org.wartremover.warts.Any"))
+  implicit val format: OFormat[ResponseModel] = Json.format
+
+  val sessionIdNum: Int = Random.between(10000, 99999)
+
+  val defaultResponse: ResponseModel =
```

Comment:
to make a random number be generated each time we call `defaultResponse` we could make `defaultResponse` a `def` perhaps and stick the `sessionIdNum` val inside the def so that it gets evaluated fresh each time `defaultResponse` is called